server/container_create_linux.go

```diff
@@ -1235,11 +1238,11 @@ func (s *Server) addOCIBindMounts(ctx context.Context, ctr ctrfactory.Container,
 }
 
 // mountImage mounts the provided imageRef using the mountLabel and returns the hostPath on success.
```

Comment:
Nit: I wonder if it would make sense to update the comment to reflect the new return value?

Reply:
Fixed as suggested :+1: